mitmproxy/addons/dumper.py

```diff
@@ -277,37 +277,37 @@ def error(self, f):
         if self.match(f):
             self.echo_flow(f)
 
-    def websocket_error(self, f):
+    def websocket_error(self, f: http.HTTPFlow):
         self.echo_error(
-            "Error in WebSocket connection to {}: {}".format(
-                human.format_address(f.server_conn.address), f.error
-            ),
+            f"Error in WebSocket connection to {human.format_address(f.server_conn.address)}: {f.error}",
             fg="red"
         )
 
-    def websocket_message(self, f):
+    def websocket_message(self, f: http.HTTPFlow):
+        assert f.websocket is not None
```

Comment:
Are these assertion necessary? If `websocket_message` is called without a websocket then the add-on core is broken, right?